api/app/services/pdfs/expenses_overview_service.rb

```diff
@@ -0,0 +1,220 @@
+      },
+      {
+        content: I18n.t('activerecord.attributes.expense_sheet.clothing'),
+        colspan: 2, background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('pdfs.expense_sheet.expense_table.row_headers.extra'),
+        background_color: COLOR_GREY, align: :center
+      },
+      {
+        content: I18n.t('pdfs.expense_sheet.expense_table.headers.full_amount'),
+        colspan: 2, background_color: COLOR_GREY, align: :right
+      }
+    ].freeze
+
+    TABLE_SUB_HEADER = [
+      {},
+      {},
+      {},
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :left },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right },
+      { content: 'Tage', background_color: COLOR_GREY, align: :right },
+      { content: 'Fr.', background_color: COLOR_GREY, align: :right }
+    ].freeze
+
+    def initialize(service_specifications, dates)
+      @beginning = dates.beginning
+      @ending = dates.ending
+      @service_specifications = service_specifications
+
+      update_font_families
+
+      header
+      content_table
+    end
+
+    def document
+      @document ||= Prawn::Document.new(page_size: 'A4', page_layout: :landscape)
+    end
+
+    private
+
+    def header
+      text I18n.t('pdfs.expenses_overview.swo', date: I18n.l(Time.zone.today)), align: :right, size: 8
+      text I18n.t('pdfs.expenses_overview.basedon', date: I18n.l(Time.zone.today)), align: :right, size: 8
+      text(
+        I18n.t(
+          'pdfs.expenses_overview.title',
+          beginning: I18n.l(@beginning),
+          ending: I18n.l(@ending)
+        ),
+        align: :left,
+        style: :bold,
+        size: 15
+      )
+    end
+
+    def content_table
+      font_size 9
+      table([TABLE_HEADER, TABLE_SUB_HEADER],
+            cell_style: { borders: %i[] },
+            width: bounds.width,
+            header: true,
+            column_widths: [40, 90, 80, 30, 35, 30, 35, 30, 35, 30, 35, 30, 35, 60, 30, 30, 40, 30]) do
+        row(0).font_style = :bold
+      end
+      @service_specifications.each do |_name, expense_sheets|
```

Comment:
Ich hetti de Block au ine neui Methode gno, demit `content_table` nid z lang wird

Reply:
Wenns en Hash isch wod nur d values wettsch, chasch au 
```suggestion
      @service_specifications.values.each do |expense_sheets|
```
neh